app/Http/Controllers/SettingsController.php

```diff
@@ -1278,6 +1278,12 @@ public function postRestore($filename = null)
                 // If it's greater than 300, it probably worked
                 $output = Artisan::output();
 
+                /* Run migrations */
+                \Log::debug('Migrating database...');
+                Artisan::call('migrate', ['--force' => true]);
+                $migrate_output = Artisan::output();
+                \Log::debug($migrate_output);
+
                 if (strlen($output) > 300) {
```

Comment:
If the `> 300` test doesn't work, then won't the stuff that re-adds your user also not work?

Reply:
Yeah, but that means the restore failed anyway. The output normally shows you the `blah was skipped`, etc.